scripts/game.js

```diff
@@ -314,6 +314,7 @@ function Game(debugMode, startLevel) {
     };
 
     this._evalLevelCode = function (allCode, playerCode, isNewLevel, restartingLevelFromScript) {
+        this.map._clearIntervals();
```

Comment:
This bug (timers keep running while level loads) has always existed, but it never had any effect until the credits level rewrite.